### ISSUE TITLE
[Feature Request] prevent Vuetify components from reacting to key events with modifiers

### ISSUE THREAD

**Issue by NothingWeAre** [vuetifyjs/vuetify#2940]
Hello.
I don't know if this should be [Bug] or [Feature Request], but it would be usefull if components, such as **VSwitch** , **VSelect** and other, will not react to key events with modifiers.
Precisely key events for `enter` and `space` with `ctrl` or `alt` modifiers. In my case there is two cases which can be resolved by such change:

* When user is focused on **VSwitch** and trying to open window menu by pressing `alt+space` **VSwitch** changes it's state
* If user focused on any component reacting to `enter` it is not possible to efficiantly implement reactions to `ctrl+enter` or `alt+enter`, for example to initiate submit of form or any other action, because som components will either change their states or display unexpected interface behaviors.

**Comment by johnleider** [comment]
Could you please present a use case that this solves?

**Comment by NothingWeAre** [comment]
We are migrating legacy app and want to enable user to submit from by using only keyboard, and without requiring user to reach submit button (by pressing `tab` multiple times).

And, since normally dropdowns react to `enter` and form is not submitted in such case, we attempted to catch any `ctrl+enter` events of parent element.
```html
<form @keydown.ctrl.enter="doSomething" @submit="doSomething">
 <v-select ......></v-select>
 <v-switch ......></v-switch>
</form>
```
but, since Vuetify components react on any event that contains `enter`, form may change state(v-switch) or produce undesired interface feedback (opening dropdowns/menus)

`alt+space` combination was reported by user and ignoring `space` and `enter` with `ctrl` and `alt` modifiers on selects/checkboxes is an actual behavior on all main browsers (IE/FF/Chrome/Safari).

 aside from **v-select** and **v-switch** same applies to **v-menu** when used as input in form.

PS. I may be missing something, but I have found no way to prevent child element from reacting to event from catcher placed on parent element.

**Comment by KaelWD** [comment]
>  I have found no way to prevent child element from reacting to event from catcher placed on parent element

`@keydown.capture.stop`

**Comment by NothingWeAre** [comment]
@KaelWD This actually works. Thanks.
I was trying different combinations of modifiers and somehow missed `.stop`, or done something incorrect.

While this does not resolve issue with `alt+space`, this is not as important, at least for us, so this issue can be closed.

**Comment by KaelWD** [comment]
The one you missed was probably `capture`, it's like bubbling but backwards. https://www.quirksmode.org/js/events_order.html

This is probably still something we can change, but I can't promise any sort of priority.

**Comment by johnleider** [comment]
Fixed in 57597a3e4fdb346650406a108f38c85cbbd43cba
